src/BuildScriptGeneratorCli/Commands/InstallRunTimeCommand.cs

```diff
@@ -59,12 +49,7 @@ internal override int Execute(IServiceProvider serviceProvider, IConsole console
 
         internal override bool IsValidInput(IServiceProvider serviceProvider, IConsole console)
         {
-            if (!Directory.Exists(InstallationPrefix))
-            {
-                console.WriteErrorLine($"Could not find the installation prefix directory '{InstallationPrefix}'.");
-                return false;
-            }
-
+            // TODO: validate?
```

Comment:
You need not override this method. I think it defaults to true?

Reply:
it does - i just wanted to leave its body here for now, in case new options are necessary